src/protocolsupport/zplatform/impl/glowstone/GlowStoneMiscUtils.java

```diff
@@ -10,6 +10,8 @@
 import javax.crypto.Cipher;
 import javax.crypto.SecretKey;
 
+import com.destroystokyo.paper.profile.ProfileProperty;
```

Comment:
???

Reply:
Ohh glowstone uses the paper api

Reply:
@7kasper Glowstone uses Paper's PlayerProfile API

https://github.com/GlowstoneMC/Glowstone/commit/0cfbb0692a4aafb282a6d68b26f780102bdec666#diff-bd59b1c2be78b091d2e887be7f414fe2

Reply:
(and you figured that out yourself before I commented that but GitHub didn't update the comments, whoops)